### ISSUE TITLE
Document Amazon Athena driver

### ISSUE THREAD

**Issue by ssedano** [sqlpad/sqlpad#1055]
Hi team,

I'd like to update the documentation to include Amazon Athena integration

Thank you

**Comment by rickbergfalk** [comment]
Definitely! Thanks @ssedano. Doc website is rendering the markdown under  `/docs`. The "home" content is in the README there https://github.com/sqlpad/sqlpad/blob/master/docs/README.md.

Let me know if you have any questions.

**Comment by ssedano** [comment]
@rickbergfalk any target date to create a new release/tag?

**Comment by rickbergfalk** [comment]
Oh right I suppose thats an important detail 😂 

I can tag a release later tonight as 6.8.0.

**Comment by rickbergfalk** [comment]
@ssedano 6.8.0 has been tagged. Docker builds should be available in 15 minutes or so.

**Comment by ssedano** [comment]
Thank you!
